spec/compiler/parser/to_s_spec.cr

```diff
@@ -108,4 +108,9 @@ describe "ASTNode#to_s" do
   expect_to_s %(1 <= 2 <= 3)
   expect_to_s %((1 <= 2) <= 3)
   expect_to_s %(1 <= (2 <= 3))
+
+  expect_to_s %(case\nwhen 1\nwhen 2\nend)
```

Comment:
A case without an expression is an `if elseif elseif` but with conditions at the same indentation level:

```crystal
if x
elsif y
elsif z
end

case
when x
when y
when z
end
```